apps/etterlatte-beregning/src/main/kotlin/beregning/regler/barnepensjon/Konstanter.kt

```diff
@@ -2,4 +2,5 @@ package no.nav.etterlatte.beregning.regler.barnepensjon
 
 import java.time.LocalDate
 
-val BP_1967_DATO: LocalDate = LocalDate.of(1967, 1, 1)
\ No newline at end of file
+val BP_1967_DATO: LocalDate = LocalDate.of(1967, 1, 1)
+val BP_2024_DATO: LocalDate = LocalDate.of(2023, 10, 1)
```

Comment:
Er dette tidspunktet barnepensjonregelverket skal gjelde fra? Det betyr i så fall at en sak med virkningstidspunkt 1. oktober vil bruke nytt regelverk?

Reply:
Ja, men vel og merke bare hvis feature toggle for nytt regelverk er aktivert

Reply:
@henrikg : Merk at det er satt til 1. oktober for å kunne iverksette mot oppdrag i DEV. Datoen må endres til 1.1.2024 senere, men oppdrag godtar ikke vedtak så langt frem i tid.